templates/partials/filters/filter-buttons.html

```diff
@@ -1,4 +1,4 @@
-<div class="filter">
+<div>
```

Comment:
Sure. Sorry, looks like I was overzealous in trying to clear out classes we weren't using anymore!